syft/generic/pointers/pointer_plan.py

```diff
@@ -131,3 +181,37 @@ def detail(worker: AbstractWorker, tensor_tuple: tuple) -> "PointerPlan":
 
     def wrap(self):
         return self
+
+    def __str__(self):
+        """Returns a string version of this pointer.
```

Comment:
since we are adding a docstring could we add a human readable def of what we expect